services/harmony/app/schemas/ogc-api-edr/1.1.0/ogc-api-edr-v1.1.0.yml

```diff
@@ -862,11 +856,34 @@ paths:
     post:
       tags:
         - Collection data queries
-      summary: Query end point for Corridor queries  of collection {collectionId} defined by a polygon
+      summary: Query end point for Corridor queries of collection {collectionId} defined by a polygon
       description: Return the data values for the Corridor defined by the query parameters
       operationId: postDataForCorridor
       parameters:
         - $ref: "#/components/parameters/collectionId"
+        - $ref: "#/components/parameters/coords"
```

Comment:
This is the POST route, we shouldn't add these parameters.

Reply:
Fixed